src/responsibleai/docker_env/docker_buildcontext/requirements.txt

```diff
@@ -0,0 +1,166 @@
+adal==1.2.7
```

Comment:
This is a huge list. They are all direct dependencies?

Reply:
it's a snapshot of a working environment with all packages version pinned, because pip dependency resolver times out otherwise. Addressing this together with the fix for timeout issue as suggested in other comments.